document the privileges required to see a graph reference in graph and database functions (#1461)
neo4j/docs-cypher: modules/ROOT/pages/functions/database.adoc

| The name of the database can only be returned if the provided element id belongs to a standard database in the DBMS.
|===

> The function will only return the name of the database if the user has either:

Reviewer: ```suggestion
The function only returns the name of the database if one or more of the following are true:
```

> The function only returns the name of the database if one or more of the following are true:

Reviewer 2: I still wonder if this should be nested in with the other `Considerations` but I don't know if that supports multiline

Author: I think I had trouble enumerating things in the considerations. But there's likely a way to do it.

Reviewer: pushed a commit to make it a considerations item:

<img width="868" height="286" alt="Screenshot from 2026-02-06 10-06-30" src="https://github.com/user-attachments/assets/2072c9ae-d1b9-4458-aace-2621c3c9382f" />

* access to the database

* the database is a standard database and the user has `DATATABASE MANAGEMENT`, `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE` or `SET DATABASE ACCESS` privileges

> * the database is a composite database and the user has `COMPOSITE DATATABASE MANAGEMENT`, `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE` or `ALTER COMPOSITE DATABASE` privileges

Reviewer: ```suggestion
* the user has access to the database

* the database is a standard database and the user has `DATATABASE MANAGEMENT`, `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE` or `SET DATABASE ACCESS` privileges

* the database is a composite database and the user has `COMPOSITE DATATABASE MANAGEMENT`, `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE` or `ALTER COMPOSITE DATABASE` privileges
```

> * the database is a composite database and the user has `COMPOSITE DATATABASE MANAGEMENT`, `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE` or `ALTER COMPOSITE DATABASE` privileges

Reviewer 2: ```suggestion
* the database is a standard database and the user has `DATABASE MANAGEMENT`, `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE` or `SET DATABASE ACCESS` privileges

* the database is a composite database and the user has `COMPOSITE DATABASE MANAGEMENT`, `CREATE COMPOSITE DATABASE`, `DROP COMPOSITE DATABASE` or `ALTER COMPOSITE DATABASE` privileges
```
Typos.  Also should this entire section be nested in as a Consideration instead of being normal size text?
